tests/src/test/scala/whisk/core/controller/test/WebActionsApiTests.scala

```diff
@@ -233,7 +238,7 @@ trait WebActionsApiBaseTests extends ControllerTestCommon with BeforeAndAfterEac
       if (actionName.path.defaultPackage) {
         Future.successful(theAction)
       } else {
-        getPackage(actionName.path.toFullyQualifiedEntityName) map (_ => theAction)
+        getPackage(actionName.path.toFullyQualifiedEntityName) map (pkg => theAction.inherit(pkg.parameters))
```

Comment:
there's no test for actually using a package binding here (if there is, it would fail since the parameter inheritance does not include the binding).